jboss-eap-config-openshift/16.0-openjdk11/added/standalone-openshift.xml

```diff
@@ -437,6 +437,14 @@
                 <stack name="tcp">
                     <transport type="TCP" socket-binding="jgroups-tcp"/>
                     <!-- ##JGROUPS_PING_PROTOCOL## -->
+                    <protocol type="org.jgroups.protocols.kubernetes.KUBE_PING" module="org.jgroups.kubernetes">
```

Comment:
why adding it?
it should be handled by the jgroups module.;

Reply:
removed